massa-bootstrap/src/lib.rs

```diff
@@ -46,13 +46,25 @@ pub struct GlobalBootstrapState {
     /// timestamp correction in milliseconds
     pub compensation_millis: i64,
 
-    /// list of network peers
-    pub peers: Option<BootstrapPeers>,
+    /// list of network and bootstrap peers, true if bootstrap
+    pub peers: Vec<(IpAddr, bool)>,
 
     /// state of the execution state
     pub execution: Option<BootstrapExecutionState>,
 }
 
+impl GlobalBootstrapState {
```

Comment:
Not sure about the naming of this trait, but more generally my logic is "if we are already in module `FooBar`, no needs to name something `FooBarState`, just State is enough" but this could be for sure tackle in another issue!

Reply:
see #2327